packages/pagination/src/elements/Pagination/Pagination.tsx

```diff
@@ -22,33 +22,34 @@ export type PAGE_TYPE = 'next' | 'page' | 'gap' | 'previous';
 
 export interface IPaginationProps extends Omit<HTMLAttributes<HTMLUListElement>, 'onChange'> {
   /**
-   * The currently selected page
+   * Sets the current page. Pages start at 1.
    */
   currentPage: number;
   /**
-   * The total number of pages available
+   * Defines the total number of pages
    */
   totalPages: number;
   /**
-   * The number of pages to pad the current page with when page gaps are
-   * displayed
+   * Sets the number of pages that appear between the current page and a gap indicator
    */
   pagePadding?: number;
   /**
-   * Position for the leading and trailing gap indicator, if needed based on
-   * current and total pages
+   * Positions the leading and trailing gap indicator, based on
+   * the current and total pages
    */
   pageGap?: number;
   /**
-   * @param {Any} currentPage - The currently selected page
+   * Handles page change events
```

Comment:
I'm wondering if there may be a more standardized way to describe an `onChange` handler since it is a common prop across many Garden components.

Here's one style that I think could work well for a developer audience. Something along the lines of:

| Component      | Description |
| ----------- | ----------- |
| Pagination      | A callback that is fired when a pagination page is changed       |
| Accordion   | A callback that is fired when an accordion panel is expanded or collapsed       |
| ColorPicker   | A callback that is fired when a color is selected        |

Alternatively, these could work too:

| Component      | Description |
| ----------- | ----------- |
| Pagination      | A callback that is fired when a pagination page state is changed       |
| Accordion   | A callback that is fired when an accordion panel's open state is changed        |
| ColorPicker   | A callback that is fired when a color picker state is changed        |

I like that these are a little bit more specific in their description. What do ya'll think?

Reply:
@hzhu please visit the prop documentation RFC for the intended grammar. I'm all for improving specificity, ideally within the bounds of the specified standard.

Reply:
In this case, I think "Handles..." is a solid choice for active voice present, third person, singular phrasing. How would you recommend building on that?

Reply:
I see, I think we can keep it as-is.